proseco/tests/test_acq.py

```diff
@@ -255,12 +256,66 @@ def test_calc_p_on_ccd():
     assert np.allclose(p_in_box, 0.625)
 
 
-def test_get_acq_catalog():
-    """Put it all together.  Regression test for selected stars"""
+def test_get_acq_catalog_19387():
+    """Put it all together.  Regression test for selected stars.  This obsid
+    actually changes out one of the initial catalog candidates.
+
+    From ipython:
+    >>> from proseco import acq
+    >>> acqs = acq.get_acq_catalog(19387)
+    >>> TEST_COLS = ('idx', 'slot', 'id', 'yang', 'zang', 'halfw', 'mag', 'p_acq')
+    >>> repr(acqs.meta['cand_acqs'][TEST_COLS]).splitlines()
+    """
+    acqs = get_acq_catalog(19387)
+    # Expected
+    exp = ['<AcqTable length=11>',
```

Comment:
@jeanconn - even though we don't care about SAUSAGE as a reference point, it is probably worthwhile to put in a couple of regression test cases like this for guide.  Especially now in refactoring and feature development mode you want to know what is changing (hopefully nothing for refactor).